Cognite.Extensions/TimeSeries/DataPointExtensions.cs

```diff
@@ -179,10 +183,32 @@ public static DataPointInsertionRequest ToInsertRequest(this IDictionary<Identit
                 var request = points.ToInsertRequest();
                 try
                 {
-                    using (CdfMetrics.Datapoints.WithLabels("create"))
+                    bool useGzip = false;
+                    if (gzipCountLimit >= 0)
```

Comment:
why do we need this check? count will always be bigger than 0, right? 🤔

Reply:
I mean checking count > gzipCountLmit should be sufficient?

Reply:
`gzipCountLimit` is the configuration option. I'd like it to be possible to disable the check entirely by letting it be negative.

Reply:
In general, either we throw an error for negative values or we handle it somehow. Disabling the check entirely if the value is invalid works well.

Reply:
I see, it is documented 😅